### PR TITLE
Project Plan Doc

### PR DESCRIPTION
**Pull request by PapaPwn**

### CODE REVIEW

File: docs/project-plan.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+### Add a Review
+A user should be able to review a course they have previously taken.
+
+### Comment on a Review
+A user should be able to comment on a review for further discussion.
+
+### Upvote a Review
+A user should be able to upvote a review.
+
+## Data Organisation [EPIC, THEME]
```
Comment (DarkPurple141) [inline docs/project-plan.md]: Wondering whether this EPIC should become part of Course Profiles/Something broader.

File: docs/zenhub.md [hunk 2]
```diff
@@ -8,7 +8,16 @@ Every sprint (assigned under milestones) we'll create issues relating to user `S
 
 This interconnectedness will allows us to properly track the progress and development of various projects, just by looking at the Zenhub board.
 
-All active issues should be connected to a sprint (milestone) at the minimum.
+### Issue Templates
+
```
Comment (DarkPurple141) [inline docs/zenhub.md]: Please note these changes I've made here.

**Comment by DarkPurple141** [comment]
This needs a rereview plz.

File: docs/project-plan.md (hunk 1 above)
Comment (PapaPwn) [reply docs/project-plan.md]: I think these points can be distributed between:
- Course Questions & Answers [EPIC, THEME]
- Course Reviews & Profiles [EPIC, THEME]

**Comment by DarkPurple141** [comment]
@Aruzal Move this over to issue #30.

**Pull request merged**